python/lsst/afw/geom/ellipses/PixelRegion.i

```diff
@@ -59,12 +59,15 @@
 
 %extend PyPixelRegionIterator {
 %pythoncode %{
-def next(self):
+def __next__(self):
     if self.atEnd():
         raise StopIteration()
     current = self.get()
     self.increment()
     return current
+
+next = __next__
```

Comment:
The problem is this is a swig object so it doesn't seem to be easy to make it inherit from something else. The `.next()` method is their for old code that has not been migrated to use the `next()` function. We can remove it at some point. Maybe we should add a deprecation decorator at some point.